packages/pnpm-sync-lib/src/interfaces.ts

```diff
@@ -1,17 +1,51 @@
 export interface IPnpmSyncCliArgs {
-  prepare: boolean,
-  lockfile: string,
-  store: string
-} 
+  prepare: boolean;
```

Comment:
@g-chao is Prettier not enabled in this repo?  We should fix that

Reply:
It isn't currently enabled. VSCode formatted files that I saved, so that's why they got prettified.